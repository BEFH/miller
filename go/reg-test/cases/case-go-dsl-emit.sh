# run_mlr --from $indir/s.dkvp --opprint put -q '@sum += $i; emit {"sum": @sum}'
run_mlr --from $indir/s.dkvp --opprint put -q '@sum[$a] += $i; emit {"sum": @sum}'
