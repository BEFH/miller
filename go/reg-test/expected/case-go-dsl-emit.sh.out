
mlr --from ./reg-test/input/s.dkvp --opprint put -q @sum[$a] += $i; emit {"sum": @sum}
sum
{
  "pan": 1
}
{
  "pan": 1,
  "eks": 2
}
{
  "pan": 1,
  "eks": 2,
  "wye": 3
}
{
  "pan": 1,
  "eks": 6,
  "wye": 3
}

